.jenkins/jenkins_pod.yml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+spec:
+  imagePullSecrets:
+    - name: talend-registry
+  containers:
+    - name: main
+      # 3.0.9-20220930152032 is the last root version of custom-builder
+      image: 'artifactory.datapwn.com/tlnd-docker-dev/talend/common/tsbi/custom-builder:3.0.9-20220930152032'
+      command: [ cat ]
+      tty: true
+      volumeMounts: [
+        { name: efs-jenkins-component-runtime-m2, mountPath: /root/.m2/repository},
+        { name: efs-jenkins-connectors-asdf, mountPath: /root/.asdf/installs, subPath: installs }
+      ]
+      resources: {requests: {memory: 6G, cpu: '4.0'}, limits: {memory: 8G, cpu: '5.0'}}
+      env:
+        - name: DOCKER_HOST
+          value: tcp://localhost:2375
+    - name: docker-daemon
+      image: artifactory.datapwn.com/docker-io-remote/docker:19.03.1-dind
```

Comment:
aren't we bump to docker 20.x?

Reply:
I just extracted the pod in a file, no docker bump. 
No info about docker version requested for jenkins pod.

Reply:
I also clarify the SKIP_OPTS variable
`// Maven package skipped elements
// Everything that is not needed for the veracode scan is excluded to reduce time`

Reply:
Other repositories are using this docker version? (meaning outside di team)

Reply:
Any team that didn't manage it are still in 18 or 19, I try the bump in 23 on one repo, seem to work but not the ticket scope I think.